utils/strings.js

```diff
@@ -1,5 +1,24 @@
 const _ = require('lodash');
 
+const reservedWords = [
```

Comment:
I think we can add `module` no ?
what happened if 
```js
const { module } = require('model');

module.exports = { };
```

Reply:
will do